### ISSUE TITLE
Please add an explanation in layman terms for the term "image registration" to the README

### ISSUE THREAD

**Issue by yurivict** [SuperElastix/elastix#757]
Currently it's not clear what does this software do from the README without specialized knowledge.

**Comment by N-Dekker** [comment]
@yurivict Thank you for submitting this issue. I hope pull request #774 commit 004260ad6101e5f96f4bace5b701d4682aefd09e has clarified the term sufficiently to you.

**Comment by yurivict** [comment]
Thank you.
